server/src/com/cloud/template/TemplateManagerImpl.java

```diff
@@ -275,6 +277,8 @@
     private StorageCacheManager cacheMgr;
     @Inject
     private EndPointSelector selector;
+    @Inject
+    private ImageStoreDao _imgStoreDao;
```

Comment:
So all other "Dao" variables in the class have a "_" before their name. I tried to keep it that way to follow convention. I've moved my variable to the same group where the other Dao variables are defiend, it should be more consistent now.